pkg/member/member_control.go

```diff
@@ -299,3 +314,24 @@ func (m *memberControl) IsLearnerPresent(ctx context.Context) (bool, error) {
 
 	return miscellaneous.CheckIfLearnerPresent(learnerCtx, cli)
 }
+
+// IsClusterScaledUp determines whether a etcd cluster is getting scale-up or not and returns a boolean
+func (m *memberControl) IsClusterScaledUp(ctx context.Context) (bool, error) {
+	clientSet, err := miscellaneous.GetKubernetesClientSetOrError()
+	if err != nil {
+		m.logger.Fatalf("failed to create clientset, %v", err)
+	}
+
+	state, err := miscellaneous.GetInitialClusterStateIfScaleup(ctx, m.logger, clientSet, m.podName, m.podNamespace)
+	if err != nil {
+		m.logger.Errorf("unable to check scale-up case :%v", err)
```

Comment:
This log is incorrect here, as we can print this message and then return `true` further down
Maybe we can update this message or remove this log altogether